migen/build/platforms/tinyfpga_a.py

```diff
@@ -0,0 +1,158 @@
+import bisect
+
+from migen import *
+from migen.build.generic_platform import *
+from migen.build.lattice import LatticePlatform
+
+_io = [
```

Comment:
What is this for?

Reply:
An oversight. I should remove it and do:

```
LatticePlatform.__init__(self, "LCMXO2-1200HC-4SG32C", [],
                                 _connectors, toolchain="diamond")
```